lib/util.js

```diff
@@ -369,6 +369,32 @@ exports.getVideoDescription = (html) => {
     new Entities().decode(exports.stripHTML(description[1])) : '';
 };
 
+/**
+ * Get video extras from html
+ * 
+ * @param {String} body
+ * @return {Object}
+ */
+const extrasRegexp = /<h4 class="title">[\s\S]*?([\s\S]*?)<\/h4>[\s\S]*?<ul.*?class=".*?watch-info-tag-list\">[\s\S]*?<li>([\s\S]*?)<\/li>/g;
+exports.getVideoExtras = (body) => {
+  let extrainfo = exports.between(body,
+    '<div id="watch-description-extras">',
+    '<div id="watch-discussion" class="branded-page-box yt-card">');
+  if (extrainfo === '') {
+    return {};
+  }
+  extrainfo = new Entities().decode(extrainfo);
+
+  const contentRegexp = /<a.*?>(.*?)<\/a>/;
+  const extras = {};
+  let match = extrasRegexp.exec(extrainfo,toString());
+  while (match) {
```

Comment:
Can do `while ((let match = extrasRegexp.exec(extrainfo)) != null) {` to save writing `.exec` twice in two lines

Reply:
This isn't working, unexpected token at `let`